GUI/QtQuick/qml/WalletProperties/DeleteWallet.qml

```diff
@@ -98,11 +98,13 @@ ColumnLayout {
             width: 260
 
             function click_enter() {
-                if (!bsApp.isWalletPasswordValid(wallet_properties_vm.walletId, password.input_text)){
-                    showError(qsTr("Password is incorrect"))
-                    init()
+                if (!wallet_properties_vm.isHardware && !wallet_properties_vm.isWatchingOnly) {
+                    if (!bsApp.isWalletPasswordValid(wallet_properties_vm.walletId, password.input_text)){
```

Comment:
Password validity is now checked internally in API, this invokable won't work properly anyway, as the password checking can be done only asynchronously, and this method implies immediate return. So I suggest to skip this check and remove it completely.